src/core/expr/fexpr_extend_remove.cc

```diff
@@ -37,7 +37,7 @@ std::string FExpr_Extend_Remove<EXTEND>::repr() const {
   std::string out = EXTEND? "extend" : "remove";
   out += '(';
   out += arg_->repr();
-  out += ", arg=";
+  out += ", ";
```

Comment:
So the `arg=` is not really needed here, because there is no keyword parameter called `arg`. The parameter is positional.